engine/engine-mobile.xcodeproj/project.pbxproj

```diff
@@ -2066,7 +2066,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cp \"$TARGET_BUILD_DIR/revsecurity.dylib\" \"$TARGET_BUILD_DIR/standalone-mobile-community.app/\"";
+			shellScript = "#cp \"$TARGET_BUILD_DIR/revsecurity.dylib\" \"$TARGET_BUILD_DIR/standalone-mobile-community.app/\"";
```

Comment:
Did you mean to leave this change in?

It might make it a bit of a pain for testing certain things from Xcode...